modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -134,6 +134,26 @@ class Instrument_List extends \NDB_Menu_Filter
             // the candidate exists before getting here.
             throw new \LorisException("Timepoint list requires a timepoint.");
         }
+
+        // Check imaging_browser permission, used to control imaging_browser link
+        // on the instrument_list main page
+        $perm = false;
```

Comment:
This seems to be missing the hasModule check that the other PR had. You should be able to get access to the LorisInstance class from `$request->getAttribute("loris")` if you need a reference to call it..

Reply:
Oh, that hasModule check is done in the BaseRouter, so I don't need to do it here, right?

Reply:
The BaseRouter checked if the `instrument_list` page being accessed exists, I thought you wanted to check if the `imaging_browser` exists and is active before adding a link to it?

Reply:
Right, it is a different module.

Reply:
@driusan Done, thanks for the remind.